grid/base/Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN PACKAGES="\
     microdnf clean all
 
 RUN mkdir -p /opt/selenium/ && \
+    SELENIUM_VERSION_DIRECTORY=${SELENIUM_MAJOR_VERSION}.${SELENIUM_MINOR_VERSION}.0 && \
```

Comment:
I think this change is relevant regardless of the 4.x version (past 4.1 at least), but `SELENIUM_MINOR_VERSION` is still set to `0` above, time to update that as well?

Reply:
I'm in process of testing 4.1.2 where the recent timeout issue is fixed. If that works well, we could be able to bump the version